GoogleTestAdapter/Core/Runners/SequentialTestRunner.cs

```diff
@@ -207,16 +207,16 @@ public static void LogExecutionError(ILogger logger, string executable, string w
             streamingParser.Flush();
 
             if (printTestOutput)
-                _logger.LogInfo($"{_threadName}<<<<<<<<<<<<<<< End of Output");
+                _logger.LogInfo(String.Format(Resources.EndOfOutputWithThreadInfo, _threadName));
 
             var consoleOutput = new List<string>();
             new TestDurationSerializer().UpdateTestDurations(streamingParser.TestResults);
-            _logger.DebugInfo(
-                $"{_threadName}Reported {streamingParser.TestResults.Count} test results to VS during test execution, executable: '{executable}'");
+            _logger.DebugInfo(String.Format(Resources.ReportedResultsToVS, _threadName, streamingParser.TestResults.Count, executable));
+
             foreach (TestResult result in streamingParser.TestResults)
             {
                 if (!_schedulingAnalyzer.AddActualDuration(result.TestCase, (int) result.Duration.TotalMilliseconds))
-                    _logger.LogWarning($"{_threadName}TestCase already in analyzer: {result.TestCase.FullyQualifiedName}");
+                    _logger.LogWarning($"{_threadName}{Resources.AlreadyInAnalyzer}{result.TestCase.FullyQualifiedName}");
```

Comment:
Why threadName and fullyQualifiedName here are not fill-ins in the localized string, whereas it is in most other places?
Generally I lean towards threadName **not** being a fill-in (this is a line prefix so there is nothing that can be done in localizing it, so it is just noise in the resources files). Maybe this should be fixed in most other strings.